scss/detail.scss

```diff
@@ -0,0 +1,37 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;700&display=swap');
+
+* {
+  font-family: 'Roboto', sans-serif;
+}
+
+.title {
+  border-bottom: 4px solid $earls-green;
+  font: bold 32px Roboto;
```

Comment:
We don't really use the shorthand for this. We separate this into `weigth`, `size` and `family` but the later is not really necessary 😄